packages/hooks/src/useAuthenticationStatusInfo.ts

```diff
@@ -0,0 +1,161 @@
+import useAccountStatus from './useAccountStatus';
+import { useIsAccountStatusPresent } from './useIsAccountStatusPresent';
+
+type TAcknowledgeStatuses = ('pending' | 'verified')[];
+type TFailedCases = ('rejected' | 'expired' | 'suspected')[];
+
+const useAuthenticationStatusInfo = () => {
```

Comment:
I believe this hook can be simplified or at least broken down into smaller chunks for better maintainability and readability.